zilencer/views.py

```diff
@@ -723,7 +724,34 @@ def remote_server_post_analytics(
             )
         batch_create_table_data(server, RemoteRealmAuditLog, remote_realm_audit_logs)
 
-    return json_success(request)
+    remote_realm_dict: Dict[str, Any] = {}
```

Comment:
It's probably worth definining a `TypedDict` for this type just so we can use the same type on both sides of the API.